services/parser-database/connector.py

```diff
@@ -20,20 +21,23 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
-    """Get keywords that relate to this article
+    """Get keywords that relate to this article (Javier's service)
```

Comment:
This comment probably is not appropriate. It should be referred to as the "keywords service"

Reply:
Yes, fixing!